File: maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int maxcount=0;
        for(auto &it : sentences)
        {
            int count=0;
            for(auto &ch : it)
                if(ch==' ')
                    count++;
            maxcount=max(maxcount, count);
        }
        return maxcount+1;
    }
};

File: maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] s) {
        int ans=0;
        for(int i =0; i<s.length; i++)
        {
            int cur=0;
            char ch[]=s[i].toCharArray();
            for(int j=0; j<ch.length; j++)
            {
                if(ch[j]==' ')
                   cur++; 
            }
            if(cur+1>ans)
                ans=cur+1;
        }
        return ans;
    }
}